.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: node_js
+addons:
```

Comment:
What is this doing?

Reply:
It is a dependency of cypress, I added it as tests kept failing. Weirdly only on this branch.

Reply:
I believe this is only needed for the GUI version of Cypress - for `test:e2e:interactive`. Not sure why this would be needed for Travis.